test/module-resolver.js

```diff
@@ -20,10 +23,10 @@ module.exports = ( request, options ) => {
 		packageFilter: ( pkg ) => {
 			//TODO: when Jest moves to resolver v2, this method will receive a second argument that points to the real package file (ie: no symlink)
 			//We can use it to determine if the package file is within the repo
-			if ( 'calypso:src' in pkg ) {
+			if ( packages.includes( pkg.name ) ) {
 				return {
 					...pkg,
-					main: pkg[ 'calypso:src' ] || pkg.module,
+					main: pkg[ 'calypso:src' ] || pkg.module || pkg.main,
```

Comment:
Can we also document why we prefer the `module` field over `main` in our monorepo packages, even if `calypso:src` is not present? The reason is rather subtle (the `dist/cjs` folder doesn't need to be present when building/testing) and is to obvious at all to a casual reader.